test/evergreen.yml

```diff
@@ -4935,6 +4935,48 @@ tasks:
         vars:
           test-name: evict-btree-1.wtperf
 
+  - name: cache-workload-dirty-trigger
```

Comment:
actually can we prepend this with keyword "perf-"? That's the convention we seem to follow and unfortunately it isn't followed for many-dhandle and prefetch tasks :-(.